### PR TITLE
Add urlbar events to namespace.yaml

### PR DESCRIPTION
**Pull request by rebecca-burwei**
Blocked by: https://github.com/mozilla/bigquery-etl/pull/4118

Not sure if I filled out the explore section correctly.

### CODE REVIEW

File: custom-namespaces.yaml [hunk 1]
```diff
@@ -711,6 +715,11 @@ firefox_desktop:
       tables:
         - table: mozdata.default_browser_agent.default_browser
   explores:
+    urlbar_events:
+      type: events_explore ???
+      views:
+        base_view: urlbar_events ???
+        extended_view: ???
```
Comment (rebecca-burwei) [inline custom-namespaces.yaml:722]: @scholtzan I'm trying to add a new derived table where 1 row represents 1 Glean event. Could you help me with the values here? I'm not sure what the valid values are and what they mean.

File: custom-namespaces.yaml [hunk 2]
```diff
@@ -650,6 +650,10 @@ experimentation:
 firefox_desktop:
   glean_app: true
   views:
+    urlbar_events_table:
+      type: table_view
+      tables:
+        - table: mozdata.firefox_desktop.urlbar_events
```
Comment (fbertsch) [inline custom-namespaces.yaml:656]:
```suggestion
    urlbar_events_table:
      type: table_view
      tables:
        - table: mozdata.firefox_desktop.urlbar_events
    urlbar_events:
      type: events_view
      tables:
        - events_table_view: urlbar_events_table
           base_table: mozdata.firefox_desktop.urlbar_events
```

Comment (fbertsch) [reply custom-namespaces.yaml:656]: Here, we need to create an `events_view` that utilizes the table view you created.

File: custom-namespaces.yaml [hunk 3]
```diff
@@ -732,6 +736,11 @@ firefox_desktop:
       tables:
         - table: mozdata.default_browser_agent.default_browser
   explores:
+    urlbar_events:
+      type: events_explore ???
+      views:
+        base_view: urlbar_events ???
+        extended_view: ???
```
Comment (fbertsch) [inline custom-namespaces.yaml:743]:
```suggestion
    urlbar_events:
      type: events_explore
      views:
        base_view: urlbar_events
        extended_view: urlbar_events_table
```

Comment (fbertsch) [reply custom-namespaces.yaml:743]: Here, the `urlbar_events` explore is of type `events_explore`, and the explore utilizes the `urlbar_events` Looker view (some of the confusion here is b/c Looker separates explores and views, and views are _also_ a thing in BQ).

Comment (rebecca-burwei) [reply custom-namespaces.yaml]: @fbertsch thanks for explaining!

**Review by fbertsch — APPROVED** [review]

**Pull request merged**